### PR TITLE
[OTE_SDK_TESTS] Increase code coverage against entities classes part2

### PR DESCRIPTION
**Pull request by 0de554K**
new file:   ote_sdk/ote_sdk/tests/entities/test_inference_parameters.py
	new file:   ote_sdk/ote_sdk/tests/entities/test_optimization_parameters.py
	new file:   ote_sdk/ote_sdk/tests/entities/test_subset.py

Getting rid from wrongly merged alien commits

### CODE REVIEW

File: ote_sdk/ote_sdk/tests/entities/test_inference_parameters.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <b>Description:</b>
+        To test InferenceParameters dataclass members
+
+        <b>Input data:</b>
+        Initialized instance of InferenceParameters class
+
+        <b>Expected results:</b>
+
+        <b>Steps</b>
+        1. Initiate InferenceParameters instance
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_inference_parameters.py]: find and fix all `initiate`

File: ote_sdk/ote_sdk/tests/entities/test_subset.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    def test_subset_magic_repr(self):
+        """
+        <b>Description:</b>
+        To test Subset __repr__ method
+
+        <b>Input data:</b>
+        Initialized instance of Subset enum
+
+        <b>Expected results:</b>
+        __repr__ method return correct string
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/entities/test_subset.py]:
```suggestion
        __repr__ method returns correct string
```

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
16:05:30  isort (ote_sdk)..........................................................Failed
16:05:30  - hook id: isort
16:05:30  - files were modified by this hook
16:05:30  
16:05:30  Fixing /home/jenkins/b/workspace/ote/precommit/ote_sdk_tests_runner/src/ote_sdk/ote_sdk/tests/entities/test_optimization_parameters.py
16:05:30  Fixing /home/jenkins/b/workspace/ote/precommit/ote_sdk_tests_runner/src/ote_sdk/ote_sdk/tests/entities/test_inference_parameters.py
16:05:30  Fixing /home/jenkins/b/workspace/ote/precommit/ote_sdk_tests_runner/src/ote_sdk/ote_sdk/tests/entities/test_subset.py
16:05:30  
16:05:30  isort (ote_cli|external).................................................Passed
16:05:32  black (ote_sdk|ote_cli)..................................................Failed
16:05:32  - hook id: black
16:05:32  - files were modified by this hook
16:05:32  
16:05:32  reformatted ote_sdk/ote_sdk/tests/entities/test_inference_parameters.py
16:05:32  reformatted ote_sdk/ote_sdk/tests/entities/test_optimization_parameters.py
16:05:32  All done! ✨ 🍰 ✨
16:05:32  2 files reformatted, 189 files left unchanged.
16:05:32  
16:05:32  black (rest).............................................................Passed
16:05:34  flake8...................................................................Passed
16:05:35  prettier.................................................................Passed
16:05:39  mypy (ote_sdk)...........................................................Passed
16:05:39  mypy (ote_cli)...........................................................Passed
16:05:40  mypy (external)..........................................................Passed
16:06:19  pylint...................................................................Passed
16:06:19  [Pipeline] }
16:06:19  ERROR: script returned exit code 1

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by 0de554K** [comment]
I've add a comment with url into internal JIRA ticket. Please take a look.

**Comment by 0de554K** [comment]
run ote_sdk tests

**Comment by 0de554K** [comment]
run ote_sdk tests

**Pull request merged**